dashboard-api/internal/storage/user.go

```diff
@@ -20,17 +19,70 @@ func NewUserRepository(db *sql.DB) *UserRepository {
 }
 
 func (r *UserRepository) Create(ctx context.Context, userID string) (types.User, error) {
-	_, err := r.db.Exec("INSERT INTO users (id) VALUES ($1)", userID)
+	q := `
```

Comment:
This seems reasonable to me, but I can't figure out where it's tested, am I not looking in the right place? Seems like a reasonable thing to have a couple simple user tests around, especially if one wants to refactor the query in the future (i.e return org id, etc.).

Reply:
I haven't tested this yet 😅 I'm still in the processes of figuring out approach (I mentioned this o. Slack)